### DISCUSSION TITLE
HMIP-PSM

### DISCUSSION THREAD

**Discussion by itn3rd77** [SukramJ/aiohomematic#57] [General]
Hi @SukramJ,

ich mal mal ne neue "Diskussion" für HMIP-PSM.

Das Gerät stellt sich in Home Assistant wie folgt dar:

![grafik](https://user-images.githubusercontent.com/2655169/143550719-a50b369c-3602-4ca4-8406-169a7c243c45.png)

Die Channels 3,4 und 5 werden als Switch erkannt. Ich vermute mal der Channel 3 ist der eignetliche "An/Aus" Schalter für die Steckdose. Die Channel 4 und 5 sollten eher nicht dargestellt werden, oder? Hier mal die Infos aus der CCU:

![grafik](https://user-images.githubusercontent.com/2655169/143551210-61ba435d-686d-4ba0-b3ba-4878ed2b7591.png)

**Comment by SukramJ** [comment]
Ich nehme mal an das du die virtuellen Kanäle in HA aktiviert hast. Oder?

**Reply by SukramJ** [reply]
Es gibt übrigens eine [Diskussion](https://github.com/danielperna84/hahomematic/discussions/17) zu virtuellen Kanälen.

**Reply by itn3rd77** [reply]
> Ich nehme mal an das du die virtuellen Kanäle in HA aktiviert hast. Oder?

Ah. Jetzt wo Du es sagst erinnere ich mich wieder. Das erklärt es. Danke!
